server/datastore/mysql/testing_utils.go

```diff
@@ -85,17 +85,17 @@ func setupReadReplica(t testing.TB, testName string, ds *Datastore, opts *Datast
 		// drop all foreign keys in the replica, as that causes issues even with
 		// FOREIGN_KEY_CHECKS=0
 		var fks []struct {
-			TableName      string `db:"table_name"`
-			ConstraintName string `db:"constraint_name"`
+			TableName      string `db:"TABLE_NAME"`
```

Comment:
This fixes the following:
```
=== CONT  TestDatastoreReplica/replica
    testing_utils.go:99:
                Error Trace:    testing_utils.go:99
                                                        asm_amd64.s:1581
                Error:          Received unexpected error:
                                missing destination name TABLE_NAME in *[]struct { TableName string "db:\"table_name\""; ConstraintName string "db:\"constraint_name\"" }
                Test:           TestDatastoreReplica/replica
```
(It looks like when running mysql 8 it expects uppercase always; by "it" I mean some component, driver or library,)